README.md

```diff
@@ -23,6 +23,30 @@ module "ecs_cluster" {
 }
 ```
 
+## Repository
+Creates an Elastic Container Registry with associated pull & push policies.
+The created pull & push policies can be used as policy attachments on AWS roles.
+
+### Available variables:
+  * [`repository_name`]: String(required): The name of the ECR repository
+  * [`repository_description`]: String(required): The human readable description of the ECR repository
```

Comment:
I wouldn't call it repo description as you use it within a description so it's really confusing. You use it more as a name